### ISSUE TITLE
Use dynamic configuration for providing Cogy host URLs

### ISSUE THREAD

**Issue by agis-** [skroutz/cogy#9]
This could be done by injecting a cog_url option in all the generated commands. Then the executable will read this option and act accordingly.

If we do this, then we can also get rid of the hardcoded URLs inside the executable and make it totally Rails-app-agnostic.

Blocked by #5.

**Comment by agis-** [comment]
Closing in favor of #24.
